mantis-runtime-loader/src/main/java/io/mantisrx/runtime/loader/config/WorkerConfigurationWritable.java

```diff
@@ -252,4 +252,9 @@ public double getNetworkBandwidthInMB() {
     public String taskExecutorAttributes() {
         return this.taskExecutorAttributesStr;
     }
+
+    @Override
+    public File getRegistrationStoreDir() {
+        return null;
+    }
```

Comment:
This will not and should not be used by the worker itself. Hence, it returns a null for the time being.